src/components/feedbackpage/FeedbackPage.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import { useState } from 'react';
+import { useRecoilValue } from 'recoil';
+import { Button } from 'react-bootstrap';
+
+import { HeaderDashboard } from '../dashboard/HeaderDashboard';
+import { darkModeState } from '../../store/DarkModeState';
+
+import './feedbackpage.scss';
+
+export const FeedbackPage = ()=>{
+  const [userRating, setUserRating] = useState(0);   
+  const [userFeedback, setUserFeedback] = useState("");
+  const darkModeStatus = useRecoilValue(darkModeState);
+  
+  async function submitToAPI(feedback) {
+    var URL = "https://tpzmoaw42e.execute-api.eu-west-1.amazonaws.com/prod/contact";
+    fetch(URL, {
+      method: "POST",
+      mode: "cors",
+      headers: {
+        "Content-Type": "application/json" 
+      },
+      body: JSON.stringify(feedback)
+    })
+    .then( response => response.json())
+    .then( ()=>{
+      alert("Thank you so much for your feedback! We will improve.");
+      setUserFeedback("");
+      setUserRating(0);
+    })
+    .catch((error)=>{
+      alert("Aww... So sorry something went wrong. Please try emailing. We'd love to hear from you!");
+    });
+  }
+
+  return (
+      <div id={'feedback-container'}>
+        <Container fluid>
+          <HeaderDashboard />
+          <div style={{color:`${darkModeStatus ? 'white' : 'black'}`}}>
+            <p id='line-1'> We value your opinion.</p>
+            <h1 id='line-2'> Please rate your experience</h1>
+            <div className='rating'>
+              {[...Array(5)].map((star,index) => {
+                  index+=1        
+                  return ( 
+                      <button
+                      id='userRating-btn'
+                      type='button'
+                      key={index}
+                      className={index <= userRating ? 'decided' : 'notDecided'}
+                      onClick={() => setUserRating(index)}
+                      >
+                      <span className='star'>&#9733;</span>
+                      </button>
+                  );    
+              })}
+            </div>
+            <h5 id='line-3'>How can we make your experience better?</h5>
+            <textarea id='feedback-textbox' value={userFeedback} onChange={(e)=>{setUserFeedback(e.target.value);}} placeholder='Type your feedback here...'/>
```

Comment:
Consider adding the attribute to their separate lines when the number of characters per line crosses 100-150. Will recommend using a formatted like prettier, though you would need to configure it with the `eslintric` configuration